<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="tomviz::CentralWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menuRecentlyOpened">
     <property name="title">
      <string>Recently Opened</string>
     </property>
    </widget>
    <addaction name="actionOpen"/>
    <addaction name="menuRecentlyOpened"/>
    <addaction name="separator"/>
    <addaction name="actionSaveState"/>
    <addaction name="actionLoadState"/>
    <addaction name="separator"/>
    <addaction name="actionSaveData"/>
    <addaction name="actionSaveScreenshot"/>
    <addaction name="actionSaveMovie"/>
    <addaction name="separator"/>
    <addaction name="actionSaveDebuggingState"/>
    <addaction name="separator"/>
    <addaction name="actionReset"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuModules">
    <property name="title">
     <string>Visualization Modules</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionPythonConsole"/>
    <addaction name="actionMacros"/>
   </widget>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>Data Transforms</string>
    </property>
    <addaction name="actionAlign"/>
    <addaction name="actionReconstruct"/>
    <addaction name="separator"/>
    <addaction name="actionClone"/>
    <addaction name="actionDeleteData"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuData"/>
   <addaction name="menuModules"/>
   <addaction name="menuView"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dockWidget">
   <property name="windowTitle">
    <string>Pipelines</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="pipelineBrowserWidget">
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,3,0,1">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Visualization Modules&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="tomviz::PipelineWidget" name="treeWidget">
       <property name="dragEnabled">
        <bool>true</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::DragOnly</enum>
       </property>
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="uniformRowHeights">
        <bool>true</bool>
       </property>
       <property name="columnCount">
        <number>2</number>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="headerDefaultSectionSize">
        <number>20</number>
       </attribute>
       <attribute name="headerMinimumSectionSize">
        <number>20</number>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string notr="true">2</string>
        </property>
       </column>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Data Transforms&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="tomviz::OperatorsWidget" name="treeWidget_2">
       <property name="rootIsDecorated">
        <bool>false</bool>
       </property>
       <property name="uniformRowHeights">
        <bool>true</bool>
       </property>
       <property name="columnCount">
        <number>1</number>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="headerDefaultSectionSize">
        <number>20</number>
       </attribute>
       <attribute name="headerMinimumSectionSize">
        <number>20</number>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="windowTitle">
    <string>Module Properties </string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="tomviz::ModulePropertiesPanel" name="modulePropertiesPanel"/>
  </widget>
  <widget class="pqCameraToolbar" name="cameraToolbar">
   <property name="windowTitle">
    <string>Camera Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="pqAxesToolbar" name="axesToolbar">
   <property name="windowTitle">
    <string>Axes Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QDockWidget" name="dockWidget_3">
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea|Qt::TopDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Animation</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="pqPVAnimationWidget" name="animationWidget"/>
  </widget>
  <widget class="pqVCRToolbar" name="vcrToolbar">
   <property name="windowTitle">
    <string>VCR Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QDockWidget" name="dockWidget_5">
   <property name="windowTitle">
    <string>Data Properties</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QScrollArea" name="scrollArea">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="tomviz::DataPropertiesPanel" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>76</width>
          <height>441</height>
         </rect>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QToolBar" name="modulesToolbar">
   <property name="windowTitle">
    <string>Visualization Modules Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionSaveDebuggingState">
   <property name="text">
    <string>&amp;Save Debugging State</string>
   </property>
   <property name="toolTip">
    <string>Save state with debugging information (advanced)</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionPythonConsole">
   <property name="text">
    <string>Python Console</string>
   </property>
  </action>
  <action name="actionMacros">
   <property name="text">
    <string>Macros</string>
   </property>
  </action>
  <action name="actionSaveData">
   <property name="text">
    <string>Save Data</string>
   </property>
  </action>
  <action name="actionSaveScreenshot">
   <property name="text">
    <string>Save Screenshot</string>
   </property>
  </action>
  <action name="actionSaveMovie">
   <property name="text">
    <string>Save Movie</string>
   </property>
  </action>
  <action name="actionClone">
   <property name="text">
    <string>Clone</string>
   </property>
  </action>
  <action name="actionAlign">
   <property name="text">
    <string>Align</string>
   </property>
  </action>
  <action name="actionReconstruct">
   <property name="text">
    <string>Reconstruct</string>
   </property>
  </action>
  <action name="actionSaveState">
   <property name="text">
    <string>Save State</string>
   </property>
  </action>
  <action name="actionLoadState">
   <property name="text">
    <string>Load State</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="text">
    <string>&amp;Reset</string>
   </property>
   <property name="toolTip">
    <string>Reset application</string>
   </property>
  </action>
  <action name="actionDeleteData">
   <property name="icon">
    <iconset resource="../ParaView3/ParaView/Qt/Widgets/Resources/QtWidgets.qrc">
     <normaloff>:/QtWidgets/Icons/pqDelete32.png</normaloff>:/QtWidgets/Icons/pqDelete32.png</iconset>
   </property>
   <property name="text">
    <string>Delete Data &amp;&amp; Modules</string>
   </property>
   <property name="toolTip">
    <string>Delete Data</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tomviz::CentralWidget</class>
   <extends>QWidget</extends>
   <header>CentralWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>tomviz::PipelineWidget</class>
   <extends>QTreeWidget</extends>
   <header>PipelineWidget.h</header>
  </customwidget>
  <customwidget>
   <class>tomviz::ModulePropertiesPanel</class>
   <extends>QWidget</extends>
   <header>ModulePropertiesPanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqCameraToolbar</class>
   <extends>QToolBar</extends>
   <header>pqCameraToolbar.h</header>
  </customwidget>
  <customwidget>
   <class>pqAxesToolbar</class>
   <extends>QToolBar</extends>
   <header>pqAxesToolbar.h</header>
  </customwidget>
  <customwidget>
   <class>pqPVAnimationWidget</class>
   <extends>QWidget</extends>
   <header>pqPVAnimationWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>pqVCRToolbar</class>
   <extends>QToolBar</extends>
   <header>pqVCRToolbar.h</header>
  </customwidget>
  <customwidget>
   <class>tomviz::OperatorsWidget</class>
   <extends>QTreeWidget</extends>
   <header>OperatorsWidget.h</header>
  </customwidget>
  <customwidget>
   <class>tomviz::DataPropertiesPanel</class>
   <extends>QWidget</extends>
   <header>DataPropertiesPanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../ParaView3/ParaView/Qt/Components/Resources/pqComponents.qrc"/>
  <include location="../ParaView3/ParaView/Qt/Core/Resources/pqCore.qrc"/>
  <include location="../ParaView3/ParaView/Qt/Widgets/Resources/QtWidgets.qrc"/>
 </resources>
 <connections/>
</ui>
